src/main/java/com/google/api/codegen/discovery/CommonDiscoveryProvider.java

```diff
@@ -45,12 +45,8 @@ public CommonDiscoveryProvider(
 
     Api api = context.getApi();
     String outputRoot =
-        "autogenerated/"
-            + api.getName()
-            + "/"
-            + api.getVersion()
-            + "/"
-            + context.getService().getDocumentation().getOverview();
+        "autogenerated/" + api.getName() + "/" + api.getVersion() + "/" + context.getApiRevision();
```

Comment:
I suspect that you haven't run `./gradlew googleJavaFormat`, which uses the format of the existing code here.